### ISSUE TITLE
Open other types then png and jpg 

### ISSUE THREAD

**Issue by IRooc** [ApoorvaJ/Papaya#19]
Opening a GIF and BMP file work fine if you type in the name in the open file dialog in windows. But the filter is set to jpg/png so you cannot select them from that window. Either add the bmp and gif extensions or allow \* as opt out extension in the file dialog for now.

**Comment by ApoorvaJ** [comment]
Yeah. Your suggestion makes sense. I dislike the current way of having to select JPG and PNG individually, too. I think I'll switch to showing all files and opening only the supported ones after selection. I'll probably disable GIFs manually because they won't be animated like a user may expect.

I will make this change after I'm done merging the crop and rotate tool branch I'm currently working on.

**Comment by IRooc** [comment]
Great! 

Lot's of programs open Gifs 'non-animated' like paint.net and lets you create and save those without taking the animation part into account. The open fine now if you select one specifically anyway.

So just let the readme state no animated-Gifs "yet"....
